### PR TITLE
feat(Post): Implement asset cover feature

### LINKED ISSUE #31: Show post cover image on home/post/posts page
[Issuehunt badges](https://issuehunt.io/r/saberland/create-portfolio/issues/31)

Use the `assets.cover` attribute.

---

<details>
<summary>
<b>IssueHunt Summary</b>
</summary>

### Backers (Total: $100.00)

- [issuehunt issuehunt](https://issuehunt.io/u/issuehunt) ($100.00)

### Submitted pull Requests
- [#124 feat(Post): Implement asset cover feature](https://issuehunt.io/r/saberland/create-portfolio/pull/124)
- [#94 feat(Post): Implement asset cover feature](https://issuehunt.io/r/saberland/create-portfolio/pull/94)
---

### Tips

- Checkout the [Issuehunt explorer](https://issuehunt.io/r/saberland/create-portfolio/) to discover more funded issues.
- Need some help from other developers? [Add your repositories](https://issuehunt.io/r/new) on IssueHunt to raise funds.
---
IssueHunt has been backed by the following sponsors. [Become a sponsor](https://issuehunt.io/membership/members)
</details>

### PR DESCRIPTION
**Pull request by Venefilyn**
fixes #31 

This adds a few things, assets cover image on posts, post, and home page. With home and posts list it shows the image cropped but centred.

Although not implemented, there's preparation to support landscape type with a new themeConfig `coverLayout` that defaults to `minimal`.

# Screenshots 
## Home page
image

## Posts list
image

## Post detail
image

cc @quaj @saltysugar

---

<details>
<summary>
<b>IssueHunt Summary</b>
</summary>

### Referenced issues

This pull request has been submitted to:
- [#31: Show post cover image on home/post/posts page](https://issuehunt.io/repos/188773840/issues/31)
---

IssueHunt has been backed by the following sponsors. [Become a sponsor](https://issuehunt.io/membership/members)
</details>

### CODE REVIEW

**Comment by egoist** [comment]
doesn't work on the demo

**Comment by SpyTec** [comment]
Woops, forgot to include demo. See latest change

**Comment by SpyTec** [comment]
Bump @egoist

File: packages/saber-theme-portfolio/src/components/PostCard.vue [hunk 1]
```diff
@@ -1,14 +1,23 @@
 <template>
-  <saber-link class="card post" :to="post.attributes.permalink">
-    <div class="post-title">
-      {{ post.attributes.title }}
+  <saber-link class="post post-card" :to="post.attributes.permalink">
+    <div v-if="post.attributes.assets" class="post-cover">
```
Comment (egoist) [inline packages/saber-theme-portfolio/src/components/PostCard.vue]: actually `post.attributes.assets` defaults to `{}` 😄

**Pull request closed without merging**